### DISCUSSION TITLE
Limit frequency of invocations if the same element is saved multiple times

### DISCUSSION THREAD

**Discussion by kruncher** [craftcms/webhooks#53] [Ideas] [answered]
### Description

It would be very useful to define a time threshold on a per hook basis such that a webhook for a specific element one time every X seconds. This would be useful to avoid excessively re-generating a static website when using Craft CMS in headless mode where content authors are saving content a lot.

**Comment by brandonkelly** [comment, marked as answer]
Webhooks 2.4.0 is out now, with a new **Initial Delay** setting, which you can use to spread out the execution of webhook requests.
